src/GrpcProtos/capture.proto

```diff
@@ -271,6 +271,16 @@ message ModuleUpdateEvent {
   ModuleInfo module = 3;
 }
 
+message TotalMemoryUsageInfo {
```

Comment:
I would also add a pid here, and populate it.

Reply:
Added a `pid` to `TotalMemoryUsageInfo` and also updated `MemoryUtils::GetTotalMemoryUsageInfo` to set up the pid.
Thanks for the suggestion.